.gitignore

```diff
@@ -132,8 +132,12 @@ packages/fxa-profile-server/BUCKET_NAME
 
 # fxa-react
 packages/fxa-react/**/*.js
```

Comment:
Tangentially related but I wonder why we need `packages/fxa-react/**/*.js` here at all, we definitely have a few JS files in `fxa-react`. 🤔

Reply:
There are some JS files that are already committed, but this is preventing any new ones from being added. Maybe that's the intention? A hint to use typescript for any new files?

Reply:
@dschom Worth filing an issue or okay to just leave it as is?

Reply:
Let's just leave as is. Maybe next time one of us is working on fxa-react we can experiment with removing it and see what happens.